doc/manual/src/installation/installing-binary.md

```diff
@@ -88,6 +88,26 @@ extension. The installer will also create `/etc/profile.d/nix.sh`.
 
 ### Linux
 
+To begin, if you are on Linux with systemd, remove the Nix daemon service:
+
+```console
+sudo systemctl stop nix-daemon.service
+sudo systemctl disable nix-daemon.socket nix-daemon.service
+sudo systemctl daemon-reload
+```
+
+Then you can remove systemd service files:
+
+```console
+sudo rm /etc/systemd/system/nix-daemon.service /etc/systemd/system/nix-daemon.socket
+```
+
+Also, the installer script uses systemd-tmpfiles (if presents) to create the socket directory. You may also want to remove the configuration for that:
+
+```console
+sudo rm /etc/tmpfiles.d/nix-daemon.conf
+```
```

Comment:
Since it's not absolutely clear from text or structure if the third instruction also only relates to systemd-based systems, what about we indent these into a list?

````suggestion
If you are on Linux with systemd:

1. Remove the Nix daemon service:

  ```console
  sudo systemctl stop nix-daemon.service
  sudo systemctl disable nix-daemon.socket nix-daemon.service
  sudo systemctl daemon-reload
  ```

  1. Remove systemd service files:

  ```console
  sudo rm /etc/systemd/system/nix-daemon.service /etc/systemd/system/nix-daemon.socket
  ```

  1. The installer script uses systemd-tmpfiles to create the socket directory.
     You may also want to remove the configuration for that:

  ```console
  sudo rm /etc/tmpfiles.d/nix-daemon.conf
  ```
````

Reply:
This makes it a lot clearer. Thanks!